src/hooks/__tests__/useImageRepository.spec.ts

```diff
@@ -0,0 +1,132 @@
+import { renderHook } from '@testing-library/react-hooks';
+import { createK8sWatchResourceMock } from '~/utils/test-utils';
+import { useImageRepository } from '../useImageRepository';
+// eslint-disable-next-line import/order
```

Comment:
nit: maybe we can simply fix the import order? or was there some reason to keep it this way?

Reply:
Sharp eyes. let me fix it.